scripts/utils/default_yargs.js

```diff
@@ -15,6 +15,12 @@ const checkNoDuplicate = function (array) {
   return new Set(array).size !== array.length
 }
 
+/**
+ * Ensure brackets (as an array) contains only unique elements.
+ *
+ * @param {any[]} argv script arguments.
```

Comment:
Shouldn't it be `object`, since it's not an array?
```suggestion
 * @param {object} argv script arguments.
```